<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:13pt;</string>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="insta_id">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(123, 123, 123)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="insta_pw">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(123, 123, 123)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:13pt;</string>
   </property>
   <property name="text">
    <string>패스워드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insta_login_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 168, 167); font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insta_id_delete_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 168, 167); font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="box_insta_bot_information">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>421</width>
     <height>381</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>379</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:13pt;</string>
     </property>
     <property name="text">
      <string>전체선택</string>
     </property>
    </widget>
    <widget class="QListView" name="list_insta_id">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>box_insta_bot_information</zorder>
  <zorder>label_2</zorder>
  <zorder>insta_id</zorder>
  <zorder>insta_pw</zorder>
  <zorder>label_3</zorder>
  <zorder>insta_login_btn</zorder>
  <zorder>insta_id_delete_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
